SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -434,10 +434,13 @@ FRepChangeState USpatialActorChannel::CreateInitialRepChangeState(TWeakObjectPtr
 		const FRepLayoutCmd& Cmd = Replicator.RepLayout->Cmds[CmdIdx];
 		const FRepChangedParent& Parent = Replicator.RepState->GetSendingRepState()->RepChangedPropertyTracker->Parents[Cmd.ParentIndex];
 
-		// Need special case here because Unreal has special handling of ReplicatedMovement, see AActor::OnRep_ReplicatedMovement
-		const bool RepActorMovement = Cmd.Type == ERepLayoutCmdType::RepMovement && Actor->GetReplicatedMovement().bRepPhysics;
+		// HACK: Need special case here because the gdk relies on RepMovement being replicated to the client to decide whether replicate physics.
```

Comment:
```suggestion
		// HACK: Need special case here because the gdk relies on the whole of the RepMovement struct being replicated to the client to decide whether to replicate physics.
```